unit-tests/live/image-quality/test-image-quality.py

```diff
@@ -84,6 +84,8 @@ def get_frames(config, laser_enabled):
     pipeline_profile = pipeline.start(config)
 
     sensor = pipeline_profile.get_device().first_depth_sensor()
+    if sensor.supports(rs.option.laser_power) and laser_enabled:
```

Comment:
1. Do you think setting it to max will help D400 devices to always pass?
2. Please consider switching conditions to save performance entering the function when not needed.
```
if laser_enabled && sensor.supports(rs.option.laser_power) :
```

Reply:
1 - I don't think so. I tried running max laser power vs default laser power, and sometimes the default had less black pixels than the max (we'd expect it to have more)
2 - will do